docs/source/concepts/InstrumentDefinitionFile.rst

```diff
@@ -958,7 +958,8 @@ The above one line of XML is shorthand notation for
 If name is specified, e.g. as name="det" in the first example, then as seen the
 <location> elements are given the 'name' plus a counter, where by default this counter starts from zero. This counter can optionally be changed by using
 attribute name-count-start, e.g. setting name-count-start="1" in the above
-example would have named the 10 <location> elements det1, det2, ..., det10.
+example would have named the 10 <location> elements det1, det2, ..., det10. Additionally, using the name-count-increment attribute, e.g setting 
```

Comment:
As per your code above, this also defaults to 1 (increment) if unspecified, should probably be mentioned here and tested for too.

Reply:
I think we should  update XML schema too